package.json

```diff
@@ -11,6 +11,12 @@
     "test": "node test/*.test.js --afoo --azoo='true' | tap-spec",
     "format": "prettier --write '{lib,test}/**/*.?s'"
   },
+  "files": [
+    "dist",
+    "lib",
```

Comment:
why `lib`?

Reply:
Wanted to explicitly keep the nconf LICENSE in the bundle and its place in the `/lib` near its source files seems like the best place

Reply:
With `/lib`

```
npm notice package size:  34.8 kB
npm notice unpacked size: 149.5 kB
```

Without `/lib`

```
npm notice package size:  22.8 kB
npm notice unpacked size: 97.8 kB
```